### PR TITLE
Commit alternative bulk inserter

### PR DESCRIPTION
**Pull request by lecchri1**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 setup(
     name='django-pyodbc-azure',
-    version='1.11.9.0+optel.2',
+    version='1.11.9.0+optel.2.1',
```
Comment (dontomm1) [inline setup.py]: Let's just use a single sequential number.

**Review by dontomm1 — COMMENTED** [review]
I am OK with this. Please just change the version number to `1.11.9.0+optel.3`

**Review by dontomm1 — APPROVED** [review]
Thank you!

**Comment by francoislefebvre** [comment]
Took a look. Missing background and MSSQL knowledge for a relevant review.

**Pull request merged**